sources/host-ctr/go.mod

```diff
@@ -3,16 +3,15 @@ module host-ctr
 go 1.12
 
 require (
-	github.com/aws/aws-sdk-go v1.40.8
-	github.com/awslabs/amazon-ecr-containerd-resolver v0.0.0-20201205004003-ec50b15a323d
-	github.com/containerd/containerd v1.4.8
-	github.com/google/uuid v1.3.0 // indirect
-	github.com/opencontainers/runtime-spec v1.0.3-0.20200929063507-e6143ca7d51d
-	github.com/opencontainers/selinux v1.8.2 // indirect
-	github.com/pelletier/go-toml v1.9.3
+	github.com/aws/aws-sdk-go v1.40.56
+	github.com/awslabs/amazon-ecr-containerd-resolver v0.0.0-20210811170403-63c50e4c3911
+	github.com/containerd/containerd v1.5.5
```

Comment:
Looks like this is still 1.5.5 instead of 1.5.7.

Reply:
Good catch. Hmm. I swore at some point I saw it got updated to 1.5.7. Somewhere after a `go mod tidy` call it just go reverted to a lower version for some reason...